coordinator/launch/start_real.launch

```diff
@@ -0,0 +1,24 @@
+<launch>
+
+<!-- without this the action server will give you an exception because it is sending the feedback to no where -->
+  <node name="fake_luigi_client" pkg="coordinator" type="fake_luigi_client.py" respawn="false" output="screen">
+  </node>
```

Comment:
fake luigi shouldn't be started when we have the real exeuction.

Reply:
why?

Reply:
because then Luigi will be running.

Reply:
then I suggest we add an argument, default to fake luigi because most of the time we are testing, okay?

Reply:
well, maybe we should have two launch files for the real world. One with fake interfaces and one with actual interfaces.

Reply:
now we are going to have 4 launch files ..... :D the idea was to get less launch files too

Reply:
True. But I think four is good. Like this:
- sim_test.launch (simulation + fake interfaces + rviz)
- sim.launch (simulation + rviz)
- real_test.launch (real world + fake interfaces + rviz)
- real.launch (real world, no fake interfaces, no rviz, used for the demo)

I think this does make sense.

Reply:
you can remove the arg or luigi and just copy two more files if you want, good luck naming the 4 launch files :D 
I'll be back in 15mins so you can already merge whatever version you like, both are okay with me